plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -182,6 +187,62 @@ class Bazel(
         }
     }
 
+    /**
+     * Collect the archive overrides from the `MODULE.bazel` file in [workingDir]. The overrides are collected with
+     * Buildozer and are returned as a map from package names to archive overrides. An empty map is returned if no
+     * override is defined.
+     */
+    private fun getArchiveOverrides(workingDir: File): Map<String, ArchiveOverride> {
+        val process = ProcessCapture(
+            BUILDOZER_COMMAND,
+            "print module_name urls integrity patches",
+            "//MODULE.bazel:%archive_override",
+            workingDir = workingDir
+        )
+
+        require(process.isSuccess) {
+            "Failed to get archive overrides from 'buildozer': ${process.stderr}"
+        }
+
+        // If an optional attribute is missing, buildozer outputs first a warning line and then the result with the
+        // value "(missing"):
+        // $ buildozer 'print integrity patch_cmds ' //MODULE.bazel:%archive_override
+        //      rule "//.:bazel-archive-override" has no attribute "patch_cmds"
+        //      sha256-3B9PcEylbj1e3Zc/mKRfBIfQ8oxonQpXuiNhEhSLGDM= (missing)
+        return process.stdout.lines().filterNot {
+            it.isEmpty() || it.contains("has no attribute")
+        }.associate { line ->
+            val results = line.split(' ', limit = 4)
+            val moduleName = results[0]
+            val urlsAsString = results[1]
+            val integrity = results[2]
+            val patchesAsString = results[3]
+            logger.info { "Archive override for module '$moduleName' is '$urlsAsString, $integrity, $patchesAsString'" }
+
+            val urls = urlsAsString.removeSurrounding("[", "]").split(',').map { URI.create(it) }
+
+            val patches = if (patchesAsString.contains(BUILDOZER_MISSING_VALUE)) {
+                null
+            } else {
+                logger.warn {
+                    "The module $moduleName has patches defined in the archive override, but ORT does not support " +
+                        "patches for source artefacts yet (see issue #8452)."
+                }
+
+                patchesAsString.removeSurrounding("[", "]").split(',')
+            }
+
+            val integrity2 = if (integrity.contains(BUILDOZER_MISSING_VALUE)) null else integrity
```

Comment:
Use Kotlin's `in` operator instead of `contains`.

Reply:
Or just inline this as

    integrity.takeUnless { BUILDOZER_MISSING_VALUE in it }